flink-docs/src/main/java/org/apache/flink/docs/configuration/ConfigOptionsDocGenerator.java

```diff
@@ -440,6 +450,12 @@ private static String toHtmlString(final OptionWithMetaInfo optionWithMetaInfo)
             }
         }
 
+        final String enumOptions =
```

Comment:
nit: maybe `enumValuesSection` instead of "Options" here and below? there are too many things called "Options" :D